global.go

```diff
@@ -0,0 +1,248 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package optimize
+
+import (
+	"fmt"
+	"math"
+	"sync"
+	"time"
+
+	"github.com/gonum/matrix/mat64"
+)
+
+// GlobalMethod is a global optimizer. Typically will require more function
+// evaluations and no sense of local convergence
+type GlobalMethod interface {
+	// Need to tell the optimizer how many can be done at once (for example random restart)
+	InitGlobal(tasks int) // can this be concurrent bool? Seems like optimizer should be told how many tasks
```

Comment:
I think that passing an int here is ok, perhaps `Settings.Concurrent` could be a bool.